src/utils.erl

```diff
@@ -83,8 +85,17 @@ disable_output() ->
 enable_output(Leader) ->
     group_leader(Leader, self()).
 
-common_postfix(Str1, Str2) ->
-    common_postfix(lists:reverse(Str1), lists:reverse(Str2), []).
+-spec common_file_postfix(filename(), filename()) -> filename().
+common_file_postfix(F1, F2) ->
+    common_file_postfix(lists:reverse(filename:split(F1)), lists:reverse(filename:split(F2)), []).
+
+-spec common_file_postfix(string(), string(), string()) -> filename().
+common_file_postfix([H1|T1], [H2|T2], Acc) when H1 =:= H2 -> common_file_postfix(T1,T2, [H1|Acc]);
```

Comment:
This could be a purely string-focused function so that the caller would apply both the filename split and filename join actions. (For the sake of separation of concerns.)

Reply:
You are right, it's way better like this.